source/index-management/index-management.rst

```diff
@@ -1136,4 +1138,6 @@ Changelog
 :2023-05-10:  Merge index enumeration and index management specs and get rid of references 
              to legacy server versions.
 :2023-05-18:  Add the search index management API.
-:2023-07-27:  Add search index management clarifications.
\ No newline at end of file
+:2023-07-27:  Add search index management clarifications.
+:2023-11-08: Clarify that ``readConcern`` and ``writeConcern`` must not be
+             applied to search index managment commands.
```

Comment:
You may add an empty line at EOF to prevent diff on unrelated line.

While on it, can you remove the double spaces:

```suggestion
:2023-07-27: Add search index management clarifications.
:2023-11-08: Clarify that ``readConcern`` and ``writeConcern`` must not be
             applied to search index managment commands.

```